lock/src/main/java/org/infinispan/lock/logging/Log.java

```diff
@@ -17,13 +18,12 @@
 @MessageLogger(projectCode = "ISPN")
 public interface Log extends BasicLogger {
    String LOCK_DELETE_MSG = "The lock was deleted.";
-//   String NODE_LEFT_MSG = "The node has left the cluster.";
    String UNLOCK_FAILED_MSG = "LOCK[%s] Unlock failed from node %s";
 
    @Message(value = LOCK_DELETE_MSG, id = 29001)
    ClusteredLockException lockDeleted();
 
-//   @Message(value = NODE_LEFT_MSG, id = 29002)
+//   @Message(value = "The node has left the cluster.", id = 29002)
```

Comment:
A bit too much effort to keep this message commented out, I'd rather just delete it.

Reply:
I'd rather not undo the effort already spent.  If you ask me, I wouldn;t have used a constant here in the first place. Also, I would have deleted all those unused/commented out messages from our Log classes. Their purpose is to prevent reuse? BS. We never add new messages in the middle of the range. Why not just delete all those and keep a comment of the LAST free id at the end of the file, where we are supposed to add new messages. Anyway, lets not OCD too much on things the are not related to the initial purpose.

Reply:
Ok